### DISCUSSION TITLE
Rust nif for secp256k1

### DISCUSSION THREAD

**Discussion by ayrat555** [izelnakri/eth#29] [Ideas]
Nice work updating deps. You can also use https://github.com/omgnetwork/ex_secp256k1 rust nif instead of `libsecp256k1` since you already added ex_keccak.

**Comment by izelnakri** [comment]
I just did ;)

**Comment by izelnakri** [comment]
I'm curious is there a way we can directly fetch compiled binary to save on container sizes + mix compile times in rustler? Might be worth doing.

**Comment by ayrat555** [comment]
> I'm curious is there a way we can directly fetch compiled binary to save on container sizes + mix compile times in rustler? Might be worth doing.

I think it's possible. But there is no ready solution
